src/sandbox/components/Grid.svelte

```diff
@@ -61,179 +93,125 @@
       width: ${(colEnd - colStart + 1) * columnWidth + 2}px;
     `
   })(copySelection);
+  $: requestAnimationFrame(() => scrollIntoView(isEditMode? highlight : selection));
```

Comment:
This will not be reactive. You removed `if (selection)` that was there before for this purpose